client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -303,7 +303,7 @@ export const AnalysisWizard: React.FC<IAnalysisWizard> = ({
     { id, name },
     { prevId, prevName }
   ) => {
-    if (id && stepIdReached < id) setStepIdReached(id as number);
+    if (id && stepIdReached < (id as number)) setStepIdReached(id as number);
```

Comment:
Why do we need this assertion?

Reply:
Because Id is mandatory for that object anyway.

Reply:
More likely for the same reason we need at the end of that line too.

Reply:
Seems as though this is coming from patternfly allowing string or number for ids here.